.integration/tests/dpc_testnet1.rs

```diff
@@ -515,8 +515,11 @@ fn test_testnet1_dpc_execute_constraints() {
     assert!(inner_circuit_cs.is_satisfied());
 
     // Generate inner snark parameters and proof for verification in the outer snark
-    let inner_snark_parameters =
-        <DPC as Testnet1Components>::InnerSNARK::setup(&InnerCircuit::blank(ledger.parameters()), &mut rng).unwrap();
+    let inner_snark_parameters = <DPC as Testnet1Components>::InnerSNARK::circuit_specific_setup(
+        &InnerCircuit::<DPC>::blank(ledger.parameters()),
```

Comment:
Explain: Now it is necessary to specify `DPC` in `InnerCircuit<DPC>`. Previously, it can be inferred from the InnerSNARK which is already defined on the circuit.